app/forms/candidate_interface/fac_invite_response_form.rb

```diff
@@ -0,0 +1,36 @@
+module CandidateInterface
+  class FacInviteResponseForm
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+
+    attr_reader :application_choice
+
+    attribute :application_form
+    attribute :invite
+    attribute :apply_for_this_course, :string
+
+    validates :apply_for_this_course, presence: true
+
+    def save
+      return false if invalid?
+
+      course_option = CourseOption.find_by(course_id: invite.course_id)
+
+      @application_choice = ApplicationChoice.create!(
+        status: 'unsubmitted',
+        current_course_option_id: course_option.id,
+        course_option_id: course_option.id,
+        application_form_id: application_form.id,
+      )
+
+      invite.update!(
+        candidate_decision: 'applied',
```

Comment:
I feel like we need another candidate_decision. They have not `applied` at this point, so it's a bit misleading. I think we need something like, `intend_to_apply` or `accepted_invite` similar.